packages/cli/src/prompts/prompt.ts

```diff
@@ -53,10 +54,21 @@ interface MethodOptions {
 }
 
 export let DISABLE_INTERACTIVITY: boolean =
+  !isStdinATerminal() ||
```

Comment:
I believe that `process.stdin` has a `isTTY` variable setted as `true`/`false` as appropriate, so you can avoid implementing the function in `L62` and just do:
```suggestion
  !process.stdin.isTTY ||
```

Reply:
:heart_eyes: Nice!